### PR TITLE
LPS-148502 Properties file SQL styling

### PR DESCRIPTION
**Pull request by ling-alan-huang**
_SF issue:_
https://issues.liferay.com/browse/LPS-148502

https://github.com/liferay/liferay-portal/commit/2916a840b645acf28c226b9c7baa2af37130b703
https://github.com/liferay/liferay-portal/commit/970abdba3620fda892f808d5cd38cb0282f30125
https://github.com/liferay/liferay-portal/commit/d3d12a8015ab1969c6554b27b8c093f2a823c5a0

The new SFer @qz1010

### CODE REVIEW

**Comment by ling-alan-huang** [comment]
ci:test:sf

**Comment by ling-alan-huang** [comment]
ci:test:relevant

**Comment by ling-alan-huang** [comment]
@qz1010 Can you check my last two commits again?

**Comment by ling-alan-huang** [comment]
ci:test:sf

File: test.properties [hunk 1]
```diff
@@ -5290,7 +6204,7 @@
         modules-integration-mysql57-jdk8
 
     test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][search-remote]=\
-        ${test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][search]} AND \
+        (${test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][search]}) AND \
```
Comment (ling-alan-huang) [inline test.properties:6207]: @joshchong from the history I guess you can help us.
Can we add parenthesis `()` here?
This case starts with `${` and ends with `}`

**Comment by ling-alan-huang** [comment]
ci:test:relevant

File: test.properties (hunk 1 above)
Comment (joshchong) [reply test.properties:6207]: sure that's fine

Comment (ling-alan-huang) [reply test.properties:6207]: Thanks.

**Comment by ling-alan-huang** [comment]
ci:forward

**Pull request closed without merging**